go.work.sum

```diff
@@ -343,7 +344,6 @@ github.com/chzyer/readline v1.5.1/go.mod h1:Eh+b79XXUwfKfcPLepksvw2tcLE/Ct21YObk
 github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1 h1:q763qf9huN11kDQavWsoZXJNW3xEE4JJyHa5Q25/sd8=
 github.com/chzyer/test v1.0.0 h1:p3BQDXSxOhOG0P9z6/hGnII4LGiEPOYBhs8asl/fC04=
 github.com/chzyer/test v1.0.0/go.mod h1:2JlltgoNkt4TW/z9V/IzDdFaMTM2JPIi26O1pF38GC8=
-github.com/cilium/ebpf v0.11.0/go.mod h1:WE7CZAnqOL2RouJ4f1uyNhqr2P4CCvXFIqdRDUgWsVs=
```

Comment:
I wonder why is it deleted? o_O

I'd expect it to be still used https://github.com/grafana/pyroscope/blob/next/ebpf/go.mod#L7

Reply:
Personally not too familiar with workspaces, but according to [stackoverflow](https://stackoverflow.com/questions/71611196/what-does-the-go-work-sum-file-track-in-go-1-18) `go.work.sum`. 

It would only contain things not contained/different from the referenced modules in `go.work`.

As `epbf/go.sum` contains this particular sum it can be removed. I also think our PR ci doesn't do a `go mod tidy` check yet on `ebpf`. That would also explain why it has been added in the first place.

Reply:
This should prevent it going forward: https://github.com/grafana/pyroscope/pull/2020